src/lit-html.ts

```diff
@@ -12,6 +12,21 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
+ /**
+  *  
+  * Main lit-html module.
+  * 
+  * Main exports:
+  * 
+  * -  [[html]] 
+  * -  [[svg]] 
+  * -  [[render]]
+  *   
+  * @module lit-html
+  * @preferred
+  */
+
+/** ignore this comment, which keeps typedoc from misplacing the module docs. */
```

Comment:
"Do not remove this comment - it keeps typedoc ..."

Reply:
👍